### ISSUE TITLE
Reported zero loss with LoRA tuning

### ISSUE THREAD

**Issue by irenelizeth** [foundation-model-stack/fms-hf-tuning#18]
When running LoRA tuning the reported loss is zero, the variable `max_seq_length` is set to the default value of 4096.

```
### Label: no complaint<|endoftext|><|endoftext|><|endoftext|><|endoftext|><|endoftext|><|endoftext|><|endoftext|> This instance will be ignored in loss calculation. Note, if this happens often, consider increasing the `max_seq_length`. 
### Label: complaint This instance will be ignored in loss calculation. Note, if this happens often, consider increasing the `max_seq_length`.
  
{'loss': 0.0, 'learning_rate': 6.716876096514944e-06, 'epoch': 2.04}
...
{'loss': 0.0, 'learning_rate': 0.0, 'epoch': 5.0}  
```

**Comment by Ssukriti** [comment]
is this still an issue? we have validated lora tuning with other datasets. If issue persists on your dataset, kindly share dataset with team on slack and re-open issue
